e2e/suites/metadataprops.test.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable max-statements */
+
+import { adminLogin, logout } from '../helpers/login';
+import { host } from '../config';
+import proxyMock from '../helpers/proxyMock';
+import insertFixtures from '../helpers/insertFixtures';
+import disableTransitions from '../helpers/disableTransitions';
+import { clearInput, selectDate, scrollTo, waitForNavigation } from '../helpers/formActions';
+
+describe('Metadata Properties', () => {
+  beforeAll(async () => {
+    await insertFixtures();
+    await proxyMock();
+    await disableTransitions();
+  });
+
+  it('should log in as admin then click the settings nav button.', async () => {
+    await adminLogin();
+    await waitForNavigation(expect(page).toClick('a', { text: 'Account settings' }));
+    expect(page.url()).toBe(`${host}/en/settings/account`);
+  });
+
+  it('should test number of available properties.', async () => {
+    await expect(page).toClick('a', { text: 'Templates' });
+    await expect(page).toClick('a', { text: 'Add template' });
+    const propertyList = await page.$$('.property-options-list li');
+    expect(propertyList.length).toBe(17);
+  });
+
+  it('should create a template with all the properties.', async () => {
+    await expect(page).toClick('a', { text: 'Templates' });
+    await expect(page).toClick('a', { text: 'Add template' });
+    await expect(page).toFill('input[name="template.data.name"]', 'All props');
+
+    const propertyAddButtons = await page.$$('.property-options-list li button');
+    //intentionaly leaving geolocation and the fields after out of the test.
```

Comment:
Could we expand this comment to explain that this is to avoid the "Violated Articles" property?

Reply:
Sure.